### PR TITLE
report message instead of throw an error

### PR DESCRIPTION
**Pull request by ShanniLi**
r @Raynos @kriskowal

### CODE REVIEW

**Comment by Raynos** [comment]
lgtm

File: index.js [hunk 1]
```diff
@@ -144,13 +146,27 @@ function main(argv, onResponse) {
     tcurl(opts);
 }
 
+function reportError(message, err) {
+    console.log(message);
```
Comment (kriskowal) [inline index.js]: console.error goes to stderr

**Comment by kriskowal** [comment]
:ok:

File: index.js (hunk 1 above)
Comment (ShanniLi) [inline index.js]: Good point. Updated.

**Pull request merged**